winpr/libwinpr/clipboard/dirent.c

```diff
@@ -0,0 +1,171 @@
+
+#include <stdio.h>
+#include "dirent.h"
+
+#include "../log.h"
+#define TAG WINPR_TAG("clipboard.synthetic.file")
+
+DIR* opendirW(const char* name)
```

Comment:
Having a `opendirW` and `opendirA` does not make sense. If you want to support `ANSI` builds on windows, do it internally here with the usual `define guards` as done for other windows builds